kawilliams8/palette-picker-api
# Server setup

#### What’s this PR do?

Adds all dependencies and configures knex for dev and test environments. Sets up basic server.js and test data. Creates migrations and seeds, and runs them for both palette_picker and palette_picker_test databases on postgreSQL.

#### Where should the reviewer start?

All files are brand new.  Go through db directory, server.js, migrations, seeds, etc.

#### How should this be manually tested?

Set up local database and run migrations and seeds to see that they are creating tables and populating them correctly. 

#### Any background context you want to provide?

Following the lesson plans so far! 

#### What are the relevant tickets?

https://app.clubhouse.io/palettepicker/story/14/knex-migrations
https://app.clubhouse.io/palettepicker/story/15/knex-seeds
https://app.clubhouse.io/palettepicker/story/16/express-server

#### Screenshots (if appropriate)
N/A
#### Questions:
Not at this time.
